### PR TITLE
Remove bamm:name attribute

### LINKED ISSUE #90: [Task] Update SDK to handle removal of bamm:name attribute from Meta Model
**Is your task related to a problem? Please describe.**
In the [sds-bamm-aspect-meta-model](https://github.com/OpenManufacturingPlatform/sds-bamm-aspect-meta-model) issue [[BUG] Remove bamm:name attribute #50](https://github.com/OpenManufacturingPlatform/sds-bamm-aspect-meta-model/issues/50) the `bamm:name` attribute is removed from the BAMM. The SDK must be updated to handle this change.

**Describe the solution you'd like**
The functionality provided by the SDK works as before using the name of the model elements as defined in the URN of the element. In case a model element is defined as an anonymous node, a synthetic name should be used.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by atextor**
Fixes #90

### CODE REVIEW

File: core/sds-aspect-meta-model-java/src/test/java/io/openmanufacturing/sds/metamodel/loader/AspectMetaModelInstantiatorTest.java [hunk 1]
```diff
@@ -43,7 +43,9 @@
 
 public class AspectMetaModelInstantiatorTest extends MetaModelInstantiatorTest {
    @ParameterizedTest
-   @EnumSource( value = TestAspect.class )
+   @EnumSource( value = TestAspect.class, mode = EnumSource.Mode.EXCLUDE, names = {
+         "ASPECT_WITH_TIME_SERIES" // This feature branch does not support bamm:AbstractProperty. When support for this is added, remove this
```
Comment (georgschmidtdumont) [inline core/sds-aspect-meta-model-java/src/test/java/io/openmanufacturing/sds/metamodel/loader/AspectMetaModelInstantiatorTest.java:47]: The PR for the `AbstractProperty` is merged, right? If so couldn't the main branch be merged into this branch and this exclusion resolved?

File: core/sds-aspect-meta-model-java/src/test/java/io/openmanufacturing/sds/metamodel/loader/CollectionInstantiatorTest.java [hunk 2]
```diff
@@ -139,6 +140,8 @@ public void testSortedSetCharacteristicInstantiationExpectSuccess( final KnownVe
 
    @ParameterizedTest
    @MethodSource( value = "versionsStartingWith2_0_0" )
+   /* This feature branch does not support bamm:AbstractProperty. When support for this is added, remove @Disabled */
+   @Disabled
```
Comment (georgschmidtdumont) [inline core/sds-aspect-meta-model-java/src/test/java/io/openmanufacturing/sds/metamodel/loader/CollectionInstantiatorTest.java:144]: See comment above on `AspectMetaModelInstantiatorTest`.

File: core/sds-aspect-model-document-generators/src/main/java/io/openmanufacturing/sds/aspectmodel/generator/json/AspectModelJsonPayloadGenerator.java [hunk 3]
```diff
@@ -82,6 +82,8 @@
 import io.openmanufacturing.sds.metamodel.datatypes.Curie;
 import io.openmanufacturing.sds.metamodel.impl.BoundDefinition;
 import io.openmanufacturing.sds.metamodel.loader.AspectModelLoader;
+import io.openmanufacturing.sds.metamodel.loader.Instantiator;
```
Comment (georgschmidtdumont) [inline core/sds-aspect-model-document-generators/src/main/java/io/openmanufacturing/sds/aspectmodel/generator/json/AspectModelJsonPayloadGenerator.java:85]: Are these imports really used?

File: core/sds-aspect-meta-model-java/src/test/java/io/openmanufacturing/sds/metamodel/loader/AspectMetaModelInstantiatorTest.java (hunk 1 above)
Comment (atextor) [reply core/sds-aspect-meta-model-java/src/test/java/io/openmanufacturing/sds/metamodel/loader/AspectMetaModelInstantiatorTest.java:47]: No, actually the feature branch for AbstractProperty is based on this branch here. It is already reverted in this branch: https://github.com/bci-oss/sds-sdk/commit/023c451be48ff013ee489f1814a0f0a22d46fc28

File: core/sds-aspect-meta-model-java/src/test/java/io/openmanufacturing/sds/metamodel/loader/CollectionInstantiatorTest.java (hunk 2 above)
Comment (atextor) [reply core/sds-aspect-meta-model-java/src/test/java/io/openmanufacturing/sds/metamodel/loader/CollectionInstantiatorTest.java:144]: See answer above

File: core/sds-aspect-model-document-generators/src/main/java/io/openmanufacturing/sds/aspectmodel/generator/json/AspectModelJsonPayloadGenerator.java (hunk 3 above)
Comment (atextor) [reply core/sds-aspect-model-document-generators/src/main/java/io/openmanufacturing/sds/aspectmodel/generator/json/AspectModelJsonPayloadGenerator.java:85]: No, this is a leftover. I removed all unused imports from the project now.

**Review by georgschmidtdumont — APPROVED** [review]

**Pull request merged**